src/mesh/canvas/CanvasMeshRenderer.js

```diff
@@ -135,12 +135,8 @@ export default class MeshSpriteRenderer
         const textureWidth = base.width;
         const textureHeight = base.height;
 
-        let u0 = uvs[index0] * base.width;
-        let u1 = uvs[index1] * base.width;
-        let u2 = uvs[index2] * base.width;
-        let v0 = uvs[index0 + 1] * base.height;
-        let v1 = uvs[index1 + 1] * base.height;
-        let v2 = uvs[index2 + 1] * base.height;
+        // eslint-disable-next-line one-var, one-var-declaration-per-line
+        let u0, u1, u2, v0, v1, v2;
```

Comment:
Could you find a way to do this without disabling lint rules? Please work with the current linting rules.